### PR TITLE
Fix "sign in" left align when langage different from english

### PR DESCRIPTION
**Pull request by sabrinapayet**
When language selected is french, "Sign in" left-align is not well displayed.

- Please make sure the below checklist is followed for Pull Requests.

- [x] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
- [ ] Tests are added where necessary
- [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/client/templates/angular/src/main/webapp/app/shared/login/_login.component.html [hunk 1]
```diff
@@ -23,7 +23,7 @@ <h4 class="modal-title" jhiTranslate="login.title">Sign in</h4>
 </div>
 <div class="modal-body">
     <div class="row">
-        <div class="col-md-4 offset-md-4">
+        <div class="col-md-4 offset-md-2">
```
Comment (kaidohallik) [inline generators/client/templates/angular/src/main/webapp/app/shared/login/_login.component.html]: bootstrap grid size is 12, but 4+2*2=8 <> 12  
maybe better is `class="col-md-6 offset-md-3"`

Comment (pascalgrimaud) [inline generators/client/templates/angular/src/main/webapp/app/shared/login/_login.component.html]: Not really beautiful with `class="col-md-6 offset-md-3"`. I only tested it in french

Comment (kaidohallik) [inline generators/client/templates/angular/src/main/webapp/app/shared/login/_login.component.html]: OK. Seems that `class="col-md-8 offset-md-2"` is better solution. With this English, French and Estonian looks good.

Comment (kaidohallik) [inline generators/client/templates/angular/src/main/webapp/app/shared/login/_login.component.html]: I'll create new PR to fix longer multi word title like Estonian

**Pull request merged**